product_docs/docs/biganimal/release/getting_started/creating_a_cluster/creating_a_dha_cluster.mdx

```diff
@@ -148,7 +148,9 @@ The following options aren't available when creating your cluster:
     
     See [Cluster networking architecture](01_cluster_networking) for more information. 
 
-1. In the **Backups** section:
+1.  To take the snapshot backups, enable **Volume Snapshots**. The snapshot backups are stored on the disk in the same region without degrading the performance. Enabling **Volume Snapshots** might increase the storage costs on your cloud service provider. By default, the snapshot backups are stored on the disk for 30 days.
```

Comment:
@nidhibhammar  I could not find this in the dev or staging environment for DHA cluster, not sure if this is a section, or a enable/disable control.